docs/userGuide/syntax/pageLayouts.mbdf

```diff
@@ -95,3 +101,37 @@ afterSetup(() => {
 </frontmatter>
 ```
 </span>
+
+#### Using Expressive Layout Templates
+
+In the `page.md` file of your layouts, it should come with the following reserved variable:
+
+<box>
+  {{ mainContentBody }}
+</box>
+
+which injects the actual page content in every page. This allows you to build layouts in different ways.
+
+{{ icon_example }}Adding statistics formula to the bottom of every page
+
+<box>
+  {{ mainContentBody }}
+  ```html
+    <panel header="Statistics Formula for the class" type="primary">
+      <img src="path_to_your_formula.png" />
+    </panel>
+  ```
+</box>
+
+Here's how the above content would appear:
```

Comment:
You might also want to mention the content of `index.md` separately, so that it is easier for the reader to understand that the `#### Main content of your page` is actually coming from `index.md`.

Reply:
Added the change